addon/ng2/utilities/get-dependent-files.ts

```diff
@@ -0,0 +1,105 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as ts from 'typescript';
+
+interface MapType {
```

Comment:
This seems more like a type, which can be declared as:
```typescript
declare type MapType = {
  [key: string]: MapType | string[];
};
```

I'm also kinda confused as of the role of this type. It's a `string => string[]` mapping with an hierarchical recursion on the type, but I don't see you use any key with `MapType`. Is it just a `string => string[]` mapping?